sportsdataverse/_codegen_runtime.py

```
@@ -0,0 +1,96 @@
+_RAW_DATA = "https://raw.githubusercontent.com/sportsdataverse/"
+
+
+def _get(url: str, params: Optional[dict] = None, **kwargs) -> Dict:
+    """GET ``url`` as JSON. Returns ``{}`` on failure. Strips ``None`` params."""
+    clean = {k: v for k, v in (params or {}).items() if v is not None}
+    resp = download(url=url, params=clean, **kwargs)
+    if resp is None:
+        return {}
+    try:
+        return resp.json()
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)